executor/adapter.go

```diff
@@ -774,6 +774,7 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool, hasMoreResults bool) {
 		indexNames = strings.Replace(fmt.Sprintf("%v", sessVars.StmtCtx.IndexNames), " ", ",", -1)
 	}
 	execDetail := sessVars.StmtCtx.GetExecDetails()
+	execDetail.LockKeysDuration = sessVars.StmtCtx.LockKeysDuration
```

Comment:
Why not assign this field in method GetExecDetails

Reply:
Better to do it in `GetExecDetails` function, thx for advice